#!/usr/bin/env python
PACKAGE = "laser_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Activate",	    bool_t,		0,  "Activate GUI",		                    False)
gen.add("Red_Min",		int_t,		0,	"Minimum amount of red required",       0,    0,    255)
gen.add("Green_Min",	int_t, 		0,	"Minimum amount of green required",     255,    0,    255)
gen.add("Blue_Min",		int_t,		0,	"Minumum amount of blue required",      0,    0,    255)
gen.add("Red_Max",      int_t,      0,  "Maximum amount of red required",       209,  0,    255)
gen.add("Green_Max",    int_t,      0,  "Maximum amount of green required",     255,  0,    255)
gen.add("Blue_Max",     int_t,      0,  "Maximum amount of blue required",      214,  0,    255)

exit(gen.generate(PACKAGE, "laser_detection", "ColorParams"))
